media-libs/loadpng/loadpng-1.5.recipe

```diff
@@ -1,33 +1,45 @@
-DESCRIPTION="
-loadpng is some glue that makes it easy to use libpng to load and save bitmaps \
-from Allegro programs.
-"
+SUMMARY="Reference library for working with PNG files"
+DESCRIPTION="Some glue that makes it easy to use libpng to load and save bitmaps from \
+Allegro programs."
 HOMEPAGE="http://tjaden.strangesoft.net/loadpng/"
-SOURCE_URI="http://tjaden.strangesoft.net/loadpng/loadpng-1.5.tar.gz"
-REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND="media-libs/allegro >= 4.4
-		media-libs/libpng >=1.2"
-MESSAGE="This might not build with libpng >= 1.4"
-CHECKSUM_MD5="5e9ecdf1762c3fca47477bf0263b96ba"
+COPYRIGHT="1997-2004 Psyk Software."
+LICENSE="Public Domain"
+SOURCE_URI="http://tjaden.strangesoft.net/loadpng/loadpng-$portVersion.tar.gz"
+REVISION="2"
```

Comment:
You can keep the REVISION at 1 since the previous version of this recipe was totally useless and didn't work.